onnxruntime/core/providers/webnn/builders/impl/argmax_min_op_builder.cc

```diff
@@ -41,9 +41,11 @@ Status ArgMaxMinOpBuilder::AddToModelBuilderImpl(ModelBuilder& model_builder,
   const auto select_last_index = helper.Get("select_last_index", 0);
 
   axis = HandleNegativeAxis(axis, input_rank);
+  emscripten::val axes = emscripten::val::array();
+  axes.call<void>("push", static_cast<uint32_t>(axis));
```

Comment:
Is `axis` being kept for temporary compat? I ask because the CR description says "Replace axis ... with axes", but it's still used (I'm fine if that's why we're keeping it - just checking).

Reply:
Oh, it means the WebNN spec, as it's `axis` previously in your personal fork. But latest spec uses `axes`. 

"Replace axis ... with axes" means replace `MLArgMinMaxOptions.axis` with `MLArgMinMaxOptions.axes`.